Orange/widgets/utils/scaling.py

```diff
@@ -103,67 +116,71 @@ def set_data(self, data, skip_if_same=False, no_data=False):
     def flip_attribute(self, attr):
         if attr.is_discrete:
             return 0
-        index = self.domain.index(attr)
         self.attribute_flip_info[attr] = 1 - self.attribute_flip_info.get(attr, 0)
         if attr.is_continuous:
             self.attr_values[attr] = [-self.attr_values[attr][1],
                                       -self.attr_values[attr][0]]
-
-        self.jittered_data[index] = 1 - self.jittered_data[index]
-        self.scaled_data[index] = 1 - self.scaled_data[index]
+        col = self.jittered_data.get_column_view(attr)[0]
+        col = 1 - col
+        col = self.scaled_data.get_column_view(attr)[0]
+        col = 1 - col
```

Comment:
These four lines have no effect, do they?

Did you mean something like this?

```
col *= -1
col += 1
```

Or (shorter, but maybe slower) `col[:] = 1 - col`.

Reply:
Yes, the latter one can consume even 4 times more time.